test/skimNtuple2016_HHbtag.cpp

```diff
@@ -4929,6 +4958,7 @@ int main (int argc, char** argv)
         if (int (iJet) == bjet2idx) continue ;
 
         TLorentzVector tlv_dummyJet(theBigTree.jets_px->at(iJet), theBigTree.jets_py->at(iJet), theBigTree.jets_pz->at(iJet), theBigTree.jets_e->at(iJet));
+        if (doSmearing) tlv_dummyJet = Smearer.getSmearedJet(tlv_dummyJet, theBigTree);
```

Comment:
Here I have the same question: is this the only point where we should shift before filling the jets_blabla branches?